docs/core/diagnostics/cli-tools/dotnet-dump.md

```diff
@@ -0,0 +1,135 @@
+---
+title: dotnet-dump - .NET Core
+description: Installing and using the dotnet-dump command-line tool.
+author: sdmaclea
+ms.author: stmaclea
+ms.date: 08/21/2019
+---
+# Dump collection and analysis utility (`dotnet-dump`)
+
+The `dotnet-dump` CLI global tool is a way to collect and analyze Windows and Linux dumps all without any native debugger involved like `lldb` on Linux. This tool is important on platforms like Alpine Linux where a fully working `lldb` isn't available. The `dotnet-dump` tool will allow you to run SOS commands to analyze crashes and the GC, but it isn't a native debugger so things like displaying native stack frames aren't supported.
+
+> [!NOTE]
+> `dotnet-dump` is not supported on macOS.
+
+## Installing `dotnet-dump`
+
+To install the latest release version of the `dotnet-dump` [NuGet package](https://www.nuget.org/packages/dotnet-dump):
+
+```bash
+$ dotnet tool install -g dotnet-dump
+You can invoke the tool using the following command: dotnet-dump
+Tool 'dotnet-dump' (version '3.0.0') was successfully installed.
+```
+
+For details and other options, see [Installing the diagnostics tools](installing.md).
+
+## Using `dotnet-dump`
+
+The next step is to collect a dump. This step can be skipped if a core dump has already been generated by the operating system or [`createdump`](https://github.com/dotnet/coreclr/blob/master/Documentation/botr/xplat-minidump-generation.md#configurationpolicy) on Linux.
+
+On Linux, the runtime version must be 3.0 or greater.
+
+```bash
+$ dotnet-dump collect --process-id 1902
+Writing minidump to file ./core_20190226_135837
+Written 98983936 bytes (24166 pages) to core file
+Complete
+```
+
+Now `analyze` the core dump. Only works on Linux for this preview.
```

Comment:
@mikem8361 - This is now Windows + Linux right?

Reply:
Yes, dotnet-dump analyze is supported on both on Windows and Linux.  I haven't updated the diagnostics repo doc.